client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -205,111 +214,118 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
-      >
-        {({
-          values,
-          handleSubmit,
-          setFieldValue,
-        }: FormikProps<{
-          sampleSizes: { [key: string]: IStringSampleSizeOption }
-        }>) => (
-          <Form data-testid="sample-size-form">
-            {sampleSizeOptions && (
-              <FormSection>
-                <FormSectionDescription>
-                  Choose the initial sample size for each contest you would like
-                  to use for Round 1 of the audit from the options below.
-                </FormSectionDescription>
-                {targetedContests.map(contest => {
-                  const currentOption = getIn(
-                    values,
-                    `sampleSizes[${contest.id}]`
-                  )
-                  return (
-                    <ElevatedCard key={contest.id}>
-                      <FormSectionDescription>
-                        <H4>{contest.name}</H4>
-                        <RadioGroup
-                          name={`sampleSizes[${contest.id}]`}
-                          onChange={e => {
-                            const selectedOption = sampleSizeOptions[
-                              contest.id
-                            ].find(c => c.key === e.currentTarget.value)
-                            setFieldValue(
-                              `sampleSizes[${contest.id}]`,
-                              selectedOption
-                            )
-                          }}
-                          selectedValue={getIn(
-                            values,
-                            `sampleSizes[${contest.id}][key]`
-                          )}
-                          disabled={locked}
-                        >
-                          {sampleSizeOptions[contest.id].map(
-                            (option: ISampleSizeOption) => {
-                              return option.key === 'custom' ? (
-                                <Radio value="custom" key={option.key}>
-                                  Enter your own sample size (not recommended)
-                                </Radio>
-                              ) : (
-                                <Radio value={option.key} key={option.key}>
-                                  {option.key === 'asn'
-                                    ? 'BRAVO Average Sample Number: '
-                                    : ''}
-                                  {`${option.size} samples`}
-                                  {option.prob
-                                    ? ` (${percentFormatter.format(
-                                        option.prob
-                                      )} chance of reaching risk limit and completing the audit in one round)`
-                                    : ''}
-                                </Radio>
+      {auditType !== 'BATCH_COMPARISON' || talliesUploadsCompleted ? (
+        <p>
+          Waiting for jurisdiction data to calculate required number of batches
```

Comment:
what about something like "All jurisdiction files must be uploaded to calculate the sample size. View jurisdiction upload progress." with "View jurisdiction upload progress" linking to the progress view

Reply:
This was the text @mcchilders provided, but I like what you suggested. @mcchilders thoughts?

Reply:
Ok that language change is fine @MorganLove!